src/ibm/hypervisor-network-mgr-src/hyp_network_manager.cpp

```diff
@@ -219,15 +219,7 @@ void HypNetworkMgr::createIfObjects()
 {
     setBIOSTableAttrs();
 
-    if (intfCount == 1)
-    {
-        // create eth0 object
-        log<level::INFO>("Creating eth0 object");
-        interfaces.emplace(
-            "eth0", std::make_shared<phosphor::network::HypEthInterface>(
-                        bus, (objectPath + "/eth0").c_str(), "eth0", *this));
-    }
-    else if (intfCount == 2)
+    if (intfCount == 1 || intfCount == 2)
```

Comment:
Please add a comment here mentioning that the hypervisor can support max 2 ethernet interfaces. Objects are created during init time to support the static network configurations on both of them